src/main/java/com/redhat/cloud/custompolicies/RouteRedirector.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.cloud.custompolicies;
+
+import io.quarkus.vertx.web.RouteFilter;
+import io.vertx.ext.web.RoutingContext;
+
+/**
+ * Redirect routes with only the major version
+ * to major.minor ones.
+ * @author hrupp
+ */
+@SuppressWarnings("unused")
+public class RouteRedirector {
+
+  private static final String API_CUSTOM_POLICIES_V_1 = "/api/custom-policies/v1/";
+  private static final String API_CUSTOM_POLICIES_V_1_0 = "/api/custom-policies/v1.0/";
+
+  /**
+   * If the requested route is the one with major version only,
+   * send a 307 "Moved temporarily" with the location of the version
+   * with major.minor.
+   * See https://tools.ietf.org/html/rfc7231#section-6.4.7
+   * Using a 308 code "Moved permanently" would make sense, but
```

Comment:
I think that 307 makes more sense, every release of a minor version would change `v1`.

> The 308 (Permanent Redirect) status code indicates that the target
   resource has been assigned a new ***permanent URI*** and any future
   references to this resource ought to use one of the enclosed URIs

https://tools.ietf.org/html/rfc7538#section-3